app/services/setting_registry.rb

```diff
@@ -53,6 +53,10 @@ def load
       end
     end
 
+    Foreman::SettingManager.settings.each do |name, opts|
+      _add(name, opts)
+    end
+
     # load all db values
     load_values
 
```

Comment:
following comment should be changed according to https://github.com/theforeman/foreman/pull/8464#discussion_r619190645 to describe better what that should mean, as here we should already decide if to create the missing settings, that are added through DSL

Reply:
I've removed the comment and went on to create the missing settings.
This is necessary for now, because we are using the DB model to update the settings.